Data/DTOs/Recruitment/CreateRecruitmentDTO.cs

```diff
@@ -11,18 +11,31 @@ namespace Data.DTOs.Recruitment
     public class CreateRecruitmentDTO
     {
         public DateTime BeginningDate { get; set; }
+
```

Comment:
Why?

Reply:
I assume you are alking about the spaces. I won't do that anymore